doc/howto/instances_manage.md

```diff
@@ -76,3 +76,21 @@ There are two ways to prevent accidental deletion of instances:
 
 - To protect a specific instance from being deleted, set [`security.protection.delete`](instance-options-security) to `true` for the instance.
   See {ref}`instances-configure` for instructions.
+
+### Rebuild an instance
+
+If you want to keep the same instance configuration, but you want to freshly start with a new instance content, you have the option to rebuild it.
+
+Once you have stopped your instance and that you ensured that the instance has no snapshots, you can proceed with the rebuild operation. It can be done in three different ways:
+
+- Rebuild the instance with its original image (the one used to create the instance).
+
+        lxc rebuild [<remote>:]<instance_name>
```

Comment:
```suggestion
        lxc rebuild <instance_name>
```
We keep the examples in the doc simple, so we leave out the [<remote>:] where it's not really relevant. We can always point to `lxc rebuild --help` for full information.